std/file.d

```diff
@@ -3081,11 +3081,19 @@ void copy(RF, RT)(RF from, RT to, PreserveAttributes preserve = preserveAttribut
 void copy(RF, RT)(auto ref RF from, auto ref RT to, PreserveAttributes preserve = preserveAttributesDefault)
     if (isConvertibleToString!RF || isConvertibleToString!RT)
 {
-    import std.map : staticMap;
+    import std.meta : staticMap;
     alias Types = staticMap!(convertToString, RF, RT);
     copy!Types(from, to, preserve);
 }
 
+unittest // issue 15319
+{
+    import std.path: dirEntries;
+    auto fs = dirEntries("", SpanMode.depth);
+    foreach(f; fs) copy(f, f);
```

Comment:
Wait, this looks like it would copy all phobos in place, which could change all mtimes. Please test this with a dedicated tempfile. Also can you check all the other occurrences of `staticMap!(convertToString`?